osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -256,8 +256,33 @@ public void PostCommand(string text, Channel target = null)
                     JoinChannel(channel);
                     break;
 
+                case "chat":
+                case "msg":
+                case "query":
+                    if (string.IsNullOrWhiteSpace(content))
+                    {
+                        target.AddNewMessages(new ErrorMessage($"Usage: /{command} [user]"));
+                        break;
+                    }
+
+                    // Check if the user has joined requested channel already.
+                    var privateChannel = JoinedChannels.FirstOrDefault(
+                        c => c.Type == ChannelType.PM && c.Users.Count == 1 && c.Name.Equals(content, StringComparison.OrdinalIgnoreCase));
+
+                    if (privateChannel != null)
+                    {
+                        CurrentChannel.Value = privateChannel;
+                        break;
+                    }
+
+                    var request = new GetUserRequest(content);
+                    request.Success += OpenPrivateChannel;
+                    request.Failure += _ => target.AddNewMessages(new ErrorMessage($"User '{content}' was not found."));
```

Comment:
That's a pretty... misleading error message? The user not existing is only one possible cause of this firing. What happens if the connection is lost, for instance?

Reply:
I assume checking the exception message is equal to `"NotFound"` will be fine, as I don't see an alternative

Reply:
I would have preferred checking the http status code (whether it's 404) but if there is no feasible way to do that then sure.